core/client/fs/src/test/java/alluxio/client/file/FileSystemContextTest.java

```diff
@@ -45,11 +37,11 @@ public void before() {
   @Test(timeout = 10000)
   public void acquireAtMaxLimit() throws Exception {
     Closer closer = Closer.create();
-
+    AlluxioConfiguration conf = Configuration.global();
     // Acquire all the clients
     FileSystemContext fsContext = FileSystemContext.create(
-        ClientContext.create(mConf));
-    for (int i = 0; i < mConf.getInt(PropertyKey.USER_FILE_MASTER_CLIENT_POOL_SIZE_MAX); i++) {
+        ClientContext.create(conf));
```

Comment:
Use `FileSystemContext.create()`, which will use `Configuration.global()` internally. And `Configuration.getInt` below.